app/controllers/analytics_controller.rb

```diff
@@ -0,0 +1,28 @@
+class AnalyticsController < AdminController
```

Comment:
Do you think a better name might be `AnalyticsBaseController`? Since it's only purpose is to be a base class?

Reply:
For consistency with ApiController and AdminController, we should keep it as AnalyticsController